components/as-ohttp-client/src/lib.rs

```diff
@@ -0,0 +1,304 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
I think you need the `extern crate rusqlite;` to make the link errors go away.  I don't understand why, but I guess this is related to the `external-sqlite` option and the `ohttp` build script.